### PR TITLE
Feature added more slang

### LINKED ISSUE #267: ABBREVIATION: add slangs
### Description

ZZZ – Sleeping, bored, tired

TIME – Tears in my eyes

BAE – Before anyone else

### PR DESCRIPTION
**Pull request by Dev-Ashank**
Closes #267

### What new changes did you make? Tick all applicable boxes
- [x] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### CODE REVIEW

File: public/server/db/rspv.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Please Respond"
```
Comment (Njong392) [inline public/server/db/rspv.json:2]: I believe you meant `rsvp`. In that case, it already exists in the records.

File: public/server/db/wuu2.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "What Are You Up To?"
```
Comment (Njong392) [inline public/server/db/wuu2.json:2]: This already exists as well.

File: public/server/db/ml.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "My Love"
```
Comment (Njong392) [inline public/server/db/ml.json:2]: This exists in our records already.

File: public/server/db/ijn.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "In Jesus Name"
```
Comment (Njong392) [inline public/server/db/ijn.json:2]: This exists in our records already.

File: public/server/db/ilysm.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "I Love You So Much"
```
Comment (Njong392) [inline public/server/db/ilysm.json:2]: This also exists.

**Review by Njong392 — CHANGES_REQUESTED** [review]

**Comment by Dev-Ashank** [comment]
Okay then i am closing this PR.

**Pull request closed without merging**